android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -214,17 +214,16 @@ private void hideToolbar() {
     }
 
     public void setNavUpButton() {
-        setNavUpButton(null);
+        setNavUpButton(null, false);
     }
 
     @SuppressWarnings({"ConstantConditions"})
-    public void setNavUpButton(Screen screen) {
+    public void setNavUpButton(Screen screen, boolean isBack) {
```

Comment:
@jskuby Why is this change necessary? Sending boolean params to functions that change what the behaviour of the function is quit an anti pattern.